js/hoverzoom.js

```diff
@@ -448,6 +448,9 @@ var hoverZoom = {
                 restoreTitles();
             }
             titledElements = img.parents('[title]').not('iframe, .lightbox, [rel^="lightbox"]');
+            // Occasionally, the img element itself will have a title attribute
+            // which we like to  handle that as well
+            titledElements = titledElements.add(img.filter('[title]');
```

Comment:
@Mattwmaster58 Thank you for the pull request. Did you actually test it? You have a syntax error on this line.

Reply:
I swear I had this working, then all of a sudden the extension wasn't loading...I figured firefox was doing something strange but this would explain it LOL